overrides/config/betterquesting/DefaultQuests.json

```diff
@@ -18642,12 +18650,13 @@
     },
     "276:10": {
       "preRequisites:11": [
-        539
+        539,
+        94
       ],
       "properties:10": {
         "betterquesting:10": {
           "autoclaim:1": 0,
-          "desc:8": "§9Phosphorus§r, §9Oxygen§r, and §9Water§r combine together into §9Phosphoric Acid§r.\n\nAlternatively, you can combine §6Phosphorous Pentoxide§r and Water for a much faster reaction.",
+          "desc:8": "§9Phosphorus§r, §9Oxygen§r, and §9Water§r combine together into §9Phosphoric Acid§r.\n\nAlternatively, you can combine §6Phosphorous Pentoxide§r and Water for a much faster reaction, or §6Apatite Dust§r and §9Sulfuric Acid§r and Water if you don\u0027t want to rely on electrolysis.",
```

Comment:
§6Apatite Dust§r and §9Sulfuric Acid§r and Water
shouldn't it be
§6Apatite Dust§r, §9Sulfuric Acid§r, and Water
?